src/moonstone-samples/src/InputSample.js

```diff
@@ -71,6 +72,21 @@ module.exports = kind({
 			]},
 			{kind: InputDecorator, disabled: true, components: [
 				{kind: RichText, disabled: true, style: 'width: 240px;'}
+			]},
+
+			{kind: Divider, content: 'Range input'},
+			{kind: InputDecorator, components: [
+				{name: 'rangeInput', kind: Input, placeholder: 'Fill out number', type: 'number', min: 25, max: 90, invalidMessage: 'Please enter a valid number.', style: 'width: 300px;', oninput:'handleInvalidInput', onchange:'handleChange'}
+			]},
+
+			{classes: 'moon-1v'},
+			{classes: 'moon-vspacing', components: [
+				{name: 'rangeValue', content: '', classes: 'moon-input-sample-value-padding'},
+				{classes: 'moon-1v'},
+				{kind: Divider, content: 'Minimum Value'},
+				{name: 'minSlider', kind: Slider, noPopup: true, showPercentage: false, enableJumpIncrement: true, constrainToBgProgress: true, value: 25, bgProgress: 90, step: 1, onChange: 'handleSliderChange'},
```

Comment:
I think it needs not to show popup for value of slider. Because it's value is displayed on another area already. 
So I added {{noPopup}} property and set to 'true'.